### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by SimonBartonPSU**
- Move the proceed button listener to [gameObjects.js](https://github.com/halyngoc/CS300-Group3-SpaceHunt/blob/master/gameObjects.js)
- Also made the move function more modular
- Wormhole check and direction check are separate functions
- Minor fixes with getElementById.

### CODE REVIEW

File: gameObjects.js [hunk 1]
```diff
@@ -76,13 +83,19 @@ function setData() {
...
 function celestialPoint() {
 
   this.planet = false;
   this.asteroid = false;
 	
 	//Incomplete object constructor for celestial points
+}
+
+//Listener for 'Proceed' button
+document.getElementById("proceed").onclick = function()
```
Comment (halyngoc) [inline gameObjects.js:97]: I might be wrong but this could be shortened to
```js
document.getElementById("proceed").onclick = spaceship.move;
```

Comment (SimonBartonPSU) [reply gameObjects.js:97]: For some reason that messes up the energy and supply value decreasing within the move function. The coordinates still update, but other stuff seems to break with that syntax change.

Comment (halyngoc) [reply gameObjects.js:97]: Huh, I wonder why.

Comment (halyngoc) [reply gameObjects.js:97]: I have a feeling this might be a bigger bug. Wanna open an issue on this and leave for later?

**Comment by SimonBartonPSU** [comment]
Closing this PR, because all of the changes are already in Liam's PR.

**Pull request closed without merging**